### PR TITLE
Adds php

### PR DESCRIPTION
**Pull request by Ankurk99**
It's just adds the basic php structure to work on.

### CODE REVIEW

**Comment by weirdwiz** [comment]
LGTM, please rebase @Ankurk99

**Comment by weirdwiz** [comment]
There still seems to be some merge conflicts :)

File: index.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+
+<head>
+<<<<<<< HEAD:index.html
```
Comment (weirdwiz) [inline index.php]: Looks like some merge conflicts crept in

Comment (weirdwiz) [reply index.php]: Resolve the conflicts and push again

File: authenticate.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?php
+session_start();
+$servername = "localhost";
+$username = "root";
+$password = "12345@Qwert";
```
Comment (weirdwiz) [inline authenticate.php]: Is it possible to use an environment variable to get the DB username and password?

Comment (Ankurk99) [reply authenticate.php]: Maybe. We also need to change servername and import the sql database. will look into this later.

**Pull request merged**